remote-method/index.js

```diff
@@ -0,0 +1,315 @@
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.httpPath == null || answers.httpPath === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'httpVerb',
+          message: 'HTTP verb:',
+          type: 'list',
+          choices: ['post', 'put', 'delete', 'get', {
+            name: '(other)',
+            value: null
+          }]
+        },
+        {
+          name: 'customHttpVerb',
+          message:
+          'Enter the custom http verb',
+          validate: validateName,
+          when: function(answers) {
+            return answers.httpVerb === null;
+          }
+        }
+      ];
+
+      var httpPath = answers.httpPath;
+
+      this.prompt(subprompts, function(answers) {
+        https.push({
+          'httpPath': httpPath,
+          'httpVerb': answers.httpVerb || answers.customHttpVerb
+        });
+        this.log('\nLet\'s add another endpoint.');
+        this.askForEndpoints();
+      }.bind(this));
+    }.bind(this));
+  },
+
+  delimAccepts: function() {
+    this.log('Let\'s add some Accept Arguments now.\n');
+  },
+
+  askForAccepts: function() {
+    var done = this.async();
+    this.log('Enter an empty name when done.');
+
+    var prompts = [
+      {
+        name: 'acceptsArg',
+        message: 'Enter name for accept argument:',
+        required: true,
+        validate: validateName
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.acceptsArg == null || answers.acceptsArg === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'acceptsType',
+          message: 'The type of accept argument:',
+          type: 'list',
+          choices: typeChoices
+        },
+        {
+          name: 'acceptsRequired',
+          message: 'Required?',
+          type: 'confirm',
+          default: false
+        },
+        {
+          name: 'acceptsDes',
+          message: 'Description of accept argument:',
+          required: true
+        },
+        {
+          name: 'httpSource',
+          message: 'Mapping to http source:',
+          default: 'body',
+          type: 'list',
+          choices: [{
+            name: '(default) No mapping',
+            value: null
+          },'body', 'context', 'query', 'res', 'req']
+        }
+      ];
+
+      var argName = answers.acceptsArg;
+
+      this.prompt(subprompts, function(answers) {
+        if (answers.httpSource) {
+          accepts.push({
+            'arg': argName,
+            'type': answers.acceptsType,
+            'required': answers.acceptsRequired,
+            'description': answers.acceptsDes,
+            'http': {source: answers.httpSource}
+          });
+        } else {
+          accepts.push({
+            'arg': argName,
+            'type': answers.acceptsType,
+            'required': answers.acceptsRequired,
+            'description': answers.acceptsDes
+          });
+        }
```

Comment:
There is no need to quote keys and please avoid code duplication.

A possible solution (feel free to use your own):

```js
var entry = {
  arg: argName,
  type: answers.acceptsType,
  required: answers.acceptsRequired,
  description: answers.acceptsDes,
};

if (answers.httpSource) {
  entry.http = { source: answers.httpSource };
}

accepts.push(entry);
```